session/src/main/java/org/apache/iotdb/session/Session.java

```diff
@@ -72,21 +72,27 @@
   private boolean isClosed = true;
   private ZoneId zoneId;
   private long statementId;
+  private int fetchSize;
 
 
   public Session(String host, int port) {
-    this(host, port, Config.DEFAULT_USER, Config.DEFAULT_PASSWORD);
+    this(host, port, Config.DEFAULT_USER, Config.DEFAULT_PASSWORD, 10000);
   }
 
   public Session(String host, String port, String username, String password) {
-    this(host, Integer.parseInt(port), username, password);
+    this(host, Integer.parseInt(port), username, password, 10000);
   }
 
   public Session(String host, int port, String username, String password) {
+    this(host, port, username, password, 10000);
```

Comment:
All these magic number '10000' should use a constant variable to replace

Reply:
move to Config in Session module